doc.html

```diff
@@ -0,0 +1,1296 @@
+                                                </p>
+                                                <p class="info_text" style="text-align:justify">
+                                                    <b>XXX pbench-cleanup</b>
+                                                    <br>clean up the pbench run directory - after this step, you will need to register any tools again.
+                                                </p>
+                                                <p class="info_text" style="text-align:justify">
+                                                    pbench-register-tool-set, pbench-register-tool and pbench-unregister-tool can also take a --remote option (see What does --remote do?) in <a href="learn.html">FAQ section</a>
+                                                    in order to allow the starting/stopping of tools and the postprocessing of results on multiple remote hosts.
+                                                </p>
+                                                <p class="info_text" style="text-align:justify">
+                                                    There is a set of miscellaneous tools for doing various and sundry things - although the name of the script indicates its purpose, if you want more information on these, you will have to read the code:
+                                                </p>
+                                                <div class="info_text" style="text-align:justify">
+                                                    <li>pbench-avg-stddev</li>
+                                                    <li>pbench-log-timestamp</li>
+                                                </div>
+                                                <p class="info_text" style="text-align:justify">
+                                                    These are used by various pieces of pbench. There is also a contrib directory that contains completely unsupported tools that various people have found useful.
+                                                </p>
+                                            </div>
+                                            <div id="secondstep" class="section_cardbody">
+                                                <p class="cardHeader">Second Steps
+                                                </p>
+                                                <p class="info_text" style="text-align:justify">
+                                                    WARNING: It is highly recommended that you use one of the pbench-< benchmark > scripts for running your benchmark. If one does not exist already, you might be able to use the pbench-user-benchmark 
+                                                    script to run your own script. The advantage is that these scripts already embody some conventions that pbench and associated tools depend on, e.g. using a timestamp in the name of the results
```

Comment:
I think any time we have "pbench" alone, it should be "Pbench".

Reply:
tried to fix!